### DISCUSSION TITLE
Security Audit

### DISCUSSION THREAD

**Discussion by ashley-OTF** [wrwrabbit/Partisan-Telegram-Android#238] [General]
Hello,

Some of [Open Technology Fund's](https://www.opentech.fund/) community members, who are at-risk activists and journalists, are interested in using this app but are concerned about security. We can't seem to find a public audit of it, so at OTF we would be happy to sponsor an audit if you're interested.

Please feel free to contact us: security@opentech.fund

Kind regards,
Ashley 
Program Specialist
Open Technology Fund

**Comment by wrwrabbit** [comment]
Hello @ashley-OTF 
Sure thing, we'd be glad to have a security audit for this project. Please let us know what is required on our end.
This repo is a branch of the official Telegram client, which I think already went through an audit, so it shouldn't be too much work to audit our branch.

**Reply by saoun** [reply]
Hi @wrwrabbit, following up from OTF here. The easiest thing to do is to submit an application through our system here: https://apply.opentech.fund/red-team-lab/

It should only take a few minutes. Once we have that, we can process the audit and connect you with one of our security firm partners. 

Let me know if you have questions.

**Reply by wrwrabbit** [reply]
Done
